scripts/send_log_config.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+from argparse import RawTextHelpFormatter
```

Comment:
Only one nit: don't import this thing additionally as this is only used at one place, replace `RawTextHelpFormatter` with `argparse.RawTextHelpFormatter` in line 20.